app/src/main/java/com/belatrixsf/allstars/ui/common/AllStarsPresenter.java

```diff
@@ -20,19 +20,23 @@
 */
 package com.belatrixsf.allstars.ui.common;
 
+import com.belatrixsf.allstars.services.AllStarsService;
+
 /**
  * @author gyosida
- *
- * AllStarsPresenter is the base clase for every presenter created
- * on the project, will hold the reference to the view and any common
- * interaction with it
+ *         AllStarsPresenter is the base clase for every presenter created
+ *         on the project, will hold the reference to the view and any common
+ *         interaction with it
  */
 public class AllStarsPresenter<T extends AllStarsView> {
 
+    protected String requestTag = AllStarsPresenter.class.getSimpleName();
```

Comment:
the variable is assigned here, but overwritten at the constructor. Should be only at the constructor, instead